src/server/game/Entities/Unit/Unit.cpp

```diff
@@ -3645,7 +3645,7 @@ void Unit::RemoveAurasDueToSpell(uint32 spellId, ObjectGuid casterGUID, uint32 r
     }
 }
 
-void Unit::RemoveAuraFromStack(uint32 spellId, ObjectGuid casterGUID, AuraRemoveMode removeMode)
+void Unit::RemoveAuraFromStack(uint32 spellId, ObjectGuid casterGUID, AuraRemoveMode removeMode, int32 num)
```

Comment:
Can auras stack more than 255 stacks? If not you could use int8. 
Just a random idea :P